tests/conf/i18n/translationresolver.js

```diff
@@ -0,0 +1,77 @@
+const TranslationResolver = require('../../../conf/i18n/translationresolver');
```

Comment:
definitely out of scope for your pr, but (cc: @tmeyer2115) do we still have plans to unify the SDK and Jambo translation classes? it feels a little weird to have double coverage for things

Reply:
On nevermind, SLAP-701 still exists!